src/amo/actions/reviews.js

```diff
@@ -63,7 +65,9 @@ export function createInternalReview(
   review: ExternalReviewType | ExternalReviewReplyType,
 ): UserReviewType {
   return {
+    addonIconUrl: review.addon.icon_url,
     addonId: review.addon.id,
+    addonName: review.addon.name,
     addonSlug: review.addon.slug,
```

Comment:
Why not exposing directly `addon: review.addon`? We had 2 fields before so that was okay, but it has just doubled here...

Reply:
True, but it still only contains a small amount of add-on information. Do you think it won't be confusing to developers who might expect a full `addon` object?

Reply:
Let's call this property `simplifiedAddon` (or `partialAddon`): https://addons-server.readthedocs.io/en/latest/topics/api/ratings.html#rating-detail-object.

Reply:
I'm not convinced this will buy us much. 

If we do this, we now have to use `simplifiedAddon.xxx` to access each property, or `const { id, name, etc } = simplifiedAddon`, each of which represents more code. It will also require changes to every component that uses add-on information from a review, whereas this patch requires no changes to any other components.

Not every consumer will need access to these two extra props, so many can remain unchanged indefinitely.

If you feel strongly about this I'll make the change, but I am not convinced of its value.

Reply:
That's what we do for `reply` already. Before, there were only 2 props to flatten, so that was okay, but now it is 4 and maybe tomorrow 5 or more. I don't really see the point of naming props `addonXXX` instead of having a single data structure for them. I see flow type definitions duplicated in this patch.

I don't know, it seems to me that it won't be future-proof.

Reply:
Fair enough. I'll make the change.